content/legal.html

```diff
@@ -3,6 +3,6 @@
 make the source code available for any binary releases you distribute.
 See the full <a href="/license">GPL 3 licence</a> for more information.</p>
 <p>The original game assets are not covered by this license and remain property of Electronic Arts Inc.<br />
-We provide asset packages based on the freeware releases of these games as a convenience to our players.</p>
-<p>Command &amp; Conquer and Command &amp; Conquer: Red Alert are trademarks or registered trademarks of Electronic&nbsp;Arts&nbsp;Inc.</p>
-</ul>
\ No newline at end of file
+We provide asset packages based on the freeware releases of these games as a convenience to our players, which we believe to be allowed by the <a href="https://www.ea.com/games/command-and-conquer/command-and-conquer-remastered/modding-faq">Command & Conquer&trade; Franchise Modding Guidelines</a>.</p>
```

Comment:
`C&amp;C Franchise Modding Guidelines`? (Mainly using `&amp;`, but this is also how `download` is formatted.)